__tests__/app.tests.js

```diff
@@ -149,3 +149,44 @@ describe("GET api/article", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET article comments with correct data type ", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        const output = res.body.comments[0];
+        expect(output.author).toEqual(expect.any(String));
+        expect(output.created_at).toEqual(expect.any(String));
+        expect(output.article_id).toEqual(expect.any(Number));
```

Comment:
Please assert against the article_id used in the parametric end point when necessary,